eventually/vote/views.py

```diff
@@ -0,0 +1,337 @@
+            return HttpResponse(status=404)
+
+        answer = Answer.create(members=data.get('members'),
+                               vote=vote,
+                               text=data.get('text'))
+        if answer:
+            answer = answer.to_dict()
+            return JsonResponse(answer, status=201)
+        return HttpResponse(status=400)
+
+    def put(self, request, team_id, event_id, vote_id, answer_id=None):  # pylint: disable=unused-argument
+        """
+        Method that handles PUT request.
+
+        :param request: the accepted HTTP request
+        :type request: HttpRequest object
+
+        :param team_id: id of the certain team
+        :type team_id: int
+
+        :param event_id: id of the certain event
+        :type event_id: int
+
+        :param vote_id: id of the certain vote
+        :type vote_id: int
+
+        :param answer_id: id of the certain answer
+        :type answer_id: int
+
+        :return: HttpResponse with status 400,404 or 201
+        """
+
+        user = request.user
+        team = Team.get_by_id(team_id)
+
+        data = request.body
+        if not data:
+            return HttpResponse(status=400)
+
+        if not answer_id:
+            return HttpResponse(status=400)
+
+        team_members = [user.id for user in team.members.all()]
+        if user.id not in team_members:
+            return HttpResponse(status=400)
+
+        answer = Answer.get_by_id(answer_id)
+        if not answer:
+            return HttpResponse(status=400)
+
+        if data.get('text') and string_validator(data['text']):
+            set_text = data.get('text')
+
+        data = {
+            'text': set_text,
+            'members': data.get("members")
+        }
+        answer.update(**data)
+        answer.to_dict()
+        return HttpResponse(answer, status=200)
+
+    def delete(self, request, team_id, event_id, vote_id, answer_id=None):  # pylint: disable=unused-argument
+        """
+        Method that handles DELETE request.
+
+        :param request: the accepted HTTP request
+        :type request: HttpRequest object
+
+        :param team_id: id of the certain team
+        :type team_id: int
+
+        :param event_id: id of the certain event
+        :type event_id: int
+
+        :param vote_id: id of the certain vote
+        :type vote_id: int
+
+        :param answer_id: id of the certain answer
+        :type answer_id: int
+
+        :return: HttpResponse with status 400,404 or 200
+        """
+        user = request.user
+        team = Team.get_by_id(team_id)
+
+        print(answer_id)
+
+        if not answer_id:
+            return HttpResponse(status=400)
+
+        team_members = [user.id for user in team.members.all()]
+        if user.id not in team_members:
+            return HttpResponse(status=400)
+
+        answer = Answer.get_by_id(answer_id)
+        if not answer:
+            return HttpResponse(status=404)
+
+        Answer.delete_by_id(answer_id)
```

Comment:
add the check if the object is successfully deleted.

Reply:
It`s not necessary